manifests/l/Lenovo/DockManager/1.5.2.2/Lenovo.DockManager.locale.en-US.yaml

```
@@ -0,0 +1,34 @@
+# Created with YamlCreate.ps1 v2.4.1 $debug=QUSU.CRLF.7-4-3.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.6.0.schema.json
+
+PackageIdentifier: Lenovo.DockManager
+PackageVersion: 1.5.2.2
+PackageLocale: en-US
+Publisher: Lenovo
+PublisherUrl: https://www.lenovo.com
+PublisherSupportUrl: https://www.lenovo.com
+PrivacyUrl: https://www.lenovo.com/privacy
+Author: Lenovo
+PackageName: Lenovo Dock Manager
+PackageUrl: https://support.lenovo.com/solutions/ht037099
+License: Copyright©Lenovo 2020.
```

```suggestion
License: Proprietary
```

A copyright isn't a license